webapp/graphite/settings.py

```diff
@@ -206,10 +207,11 @@
 FLUSHRRDCACHED = ''
 
 ## Load our local_settings
+SETTINGS_MODULE = os.environ['GRAPHITE_SETTINGS_MODULE']
```

Comment:
You should use `get()` or wrap up this in try...except - it breaks tests currently.

Reply:
Thanks, Denis.
Fixed.